frontend/src/wallets/near.js

```diff
@@ -17,13 +20,11 @@ export class Wallet {
    * @constructor
    * @param {Object} options - the options for the wallet
    * @param {string} options.networkId - the network id to connect to
-   * @param {string} options.createAccessKeyFor - the contract to create an access key for
    * @example
-   * const wallet = new Wallet({ networkId: 'testnet', createAccessKeyFor: 'contractId' });
+   * const wallet = new Wallet({ networkId: 'testnet' });
```

Comment:
```suggestion
   * const wallet = new Wallet({ networkId: 'testnet', createAccessKeyFor: 'contractId' });
```

lets bring `createAccessKeyFor` back here, as it is a valid case (people might want to add a key for their specific project